frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePullBranchAction.java

```diff
@@ -93,11 +94,11 @@ public void actionPerformed(AnActionEvent anActionEvent) {
 
       boolean isUpToDate = FetchUpToDateTimeoutStatus.isUpToDate(gitRepository);
       String fetchNotificationPrefix = isUpToDate
-          ? getString("action.GitMachete.BasePullBranchAction.notification.title.no-fetch-perform")
+          ? getNonHtmlString("action.GitMachete.BasePullBranchAction.notification.title.no-fetch-perform")
               .format(FETCH_ALL_UP_TO_DATE_TIMEOUT_AS_STRING)
-          : getString("action.GitMachete.BasePullBranchAction.notification.title.fetch-perform");
+          : getNonHtmlString("action.GitMachete.BasePullBranchAction.notification.title.fetch-perform");
       Runnable fastForwardRunnable = () -> FastForwardMerge.perform(project, gitRepository, mergeProps,
-          fetchNotificationPrefix);
+          fetchNotificationPrefix + "<br/>");
```

Comment:
Huh okay so `fetchNotificationPrefix` does ultimately accept HTML... or not? 🤔 Since it's marked as `@Untainted`... but here `<br/>` is smuggled

Reply:
here is strange situation. fetchNotificationPrefix can have html tags when it goes to successNotification  but when it goes to failNotification it cannot (fail notification is different in two places and adding html is allowed in one place and not allowed in the other). So I decided to make rule that fetchNotificationPrefix cannot have html tags inside

Reply:
> So I decided to make rule that fetchNotificationPrefix cannot have html tags inside

Okay, makes sense indeed... but here you still pass a `<br>` (i.e. a **tainted** string) in `fetchNotificationPrefix` param of `FastForwardMerge.perform`, right? So against this rule ☹️

Reply:
(pls Resolve the outdated/resolved comments so that we can see what's really still left!)